squid-exporter/README.md

```diff
@@ -0,0 +1,18 @@
+# squid-exporter
+Prometheus exporter for squid
+
+## Description
+squid-exporter converts squid counters and service_times to prometheus metrics.
+
+## Usage
+```
+    ./squid-exporter -squid-host localhost -squid-port 3128 -metrics-port 8080
+```
+
+## Option
+| option        | default   | description         |
+| ----          | ----      | ----                |
+| -squid-host   | localhost | squid host          |
+| -squid-port   | 3128      | squid port          |
+| -metrics-port | 9100      | metrics expose port |
+
```

Comment:
I added a squid-exporter entry in maintenance.md.

Reply:
Since there are multiple places(Makefile, pod.yaml) where the squid container image is used, I thought it would be good if the squid item in maintenance.md stated that the squid-exporter uses the squid image and the manifest needs to be updated.

e.g. https://github.com/cybozu/neco-containers/blob/main/maintenance.md?plain=1#L242

Reply:
Thank you. I added these NOTION and step of updating squid version in `Makefile` and `pod.yaml`